### PR TITLE
science_and_tech.json added 

### PR DESCRIPTION
**Pull request by LilShiningMan**
```SUMMARY: Infrastructure "added science_and_tech and monster_drops_advtech lists"```

#### Summary
added two new item list .jsons

#### Purpose of change
decluttering continues

#### Describe the solution
created science_and_tech and monster_drops_advtech.json files; moved appropriate lists from item_groups.json 

#### Testing
delinted; debug spawned lists.

#### Additional context
I sometimes have difficulty categorizing spawn lists in appropriate .jsons. Initially I had all of the bots and turret lists in science_and_tech. While they are advanced technology, of course these lists are what can be salvaged from them and so I figured there's enough entries to warrant them having their own .json file.

### CODE REVIEW

**Comment by LilShiningMan** [comment]
I'm baffled as to why that comment from the fridge group is listed as removed. I updated my game and my repository before making my changes and I didn't touch that line.

File: data/json/itemgroups/item_groups.json [hunk 1]
```diff
@@ -510,7 +510,6 @@
   {
     "type": "item_group",
     "id": "fridge",
```
Comment (anothersimulacrum) [inline data/json/itemgroups/item_groups.json]:
```suggestion
     "id": "fridge",
     "//": "This is a terrible item group and should be phased out.  See SUS_fridge for a more modern take on things.",
```

**Comment by LilShiningMan** [comment]
Thanks anothersimulacrum. Github and I are _not_ friends.

File: data/json/itemgroups/item_groups.json [hunk 2]
```diff
@@ -509,8 +509,8 @@
   },
   {
     "type": "item_group",
-    "id": "fridge",
-    "//": "This is a terrible item group and should be phased out.  See SUS_fridge for a more modern take on things.",
+     "id": "fridge",
```
Comment (anothersimulacrum) [inline data/json/itemgroups/item_groups.json]:
```suggestion
    "id": "fridge",
```

File: data/json/itemgroups/item_groups.json [hunk 3]
```diff
@@ -509,8 +509,8 @@
...
+     "//": "This is a terrible item group and should be phased out.  See SUS_fridge for a more modern take on things.",
```
Comment (anothersimulacrum) [inline data/json/itemgroups/item_groups.json]:
```suggestion
    "//": "This is a terrible item group and should be phased out.  See SUS_fridge for a more modern take on things.",
```

**Comment by LilShiningMan** [comment]
Wondering if I could ask for more guidance. I've never had a clang error before, but this is what I'm getting 

clang error

**Comment by anothersimulacrum** [comment]
That's not your fault - it's the package manager failing to install something.
There's not much you can do.

**Comment by kevingranade** [comment]
I'm not a huge fan of this, it doesn't seem to make things meaningfully easier to find and it adds a bunch of clutter to the commit history.

**Comment by LilShiningMan** [comment]
OK, thanks for the info, I'll just wait it out

**Pull request merged**